packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,657 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from 'react';
+import cx from 'classnames';
+import {
+  useSafeContext,
+  SvgMore,
+  type PolymorphicComponentProps,
+  type PolymorphicForwardRefComponent,
```

Comment:
can you move this into a separate `import type` statement?

see https://github.com/iTwin/iTwinUI-react/pull/996

Reply:
I see. Updated